src/routes/tools/annual-averages/ExploreData.svelte

```diff
@@ -47,17 +48,49 @@
   let showDownload = false;
   let showShare = false;
   let showChangeLocation = false;
+  let showLearnMore = false;
 
   let ChangeLocation;
   let DownloadChart;
   let ShareLink;
+  let LearnMore;
```

Comment:
It's a small thing, but because this variable eventually references the `LearnMoreModal` component it might be good to call it that to avoid confusion.